x/paychan/model.go

```diff
@@ -62,54 +61,13 @@ func (pc PaymentChannel) Copy() orm.CloneableData {
 	}
 }
 
-// PaymentChannelBucket is a wrapper over orm.Bucket that ensures that only
-// PaymentChannel entities can be persisted.
-type PaymentChannelBucket struct {
```

Comment:
Looking at the usage, I was wondering where the ModelBucket win was.

Then I see it here, where all boilerplate is taken away. Makes Bucket creation easy